docs/source/graphs.md

```diff
@@ -4,7 +4,9 @@ sidebar_title: 'graph'
 description: 'Publish and retrieve your API schema'
 ---
 
-> These commands are for graphs that do _not_ use [federation](https://www.apollographql.com/docs/federation/). When working with a federated graph, instead use the [`subgraph` comamand](./subgraphs).
+These Rover commands are _primarily_ for interacting with monolithic graphs that do _not_ use [federation](https://www.apollographql.com/docs/federation/). However, you can also use [`rover graph introspect`](#fetching-via-introspection) to fetch a federated gateway's API schema.
+
```

Comment:
and `rover graph fetch` to get the API schema from studio instead of via introspection